src/util/psbt/map/global.rs

```diff
@@ -212,9 +212,7 @@ impl Map for Global {
                         entry.insert((fingerprint1, derivation1));
                         continue
                     }
-                    return Err(psbt::Error::MergeConflict(format!(
-                        "global xpub {} has inconsistent key sources", xpub
```

Comment:
Not sure if it is intentional but we lost this part of the error message.

Reply:
We do not: we have a dedicated error type which may happen only for a global xpub (and we embed xpub into error)